### PR TITLE
spnegi - Parse Excel and create/update records in ServiceNow

### PR DESCRIPTION
**Pull request by spnegi**
This excel parser code helps reading the excel file. One can read excel columns and rows via this code. This code helps reading each row and putting validation on excel if required. One can insert or update the records in ServiceNow based on this code.

We can decide base don what column of excel we need to insert or update the record. We can also set the parent vs child relationship based on excel parser.

This is really beneficial when bulk data needs to be uploaded in system.

### CODE REVIEW

File: Flow Actions/Excel Parser/Excel Parser.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        parser.setNullToEmpty(true);
+        parser.parse(attachmentStream);
+        var headers = parser.getColumnHeaders();
+        //print headers
+
+        var rows_info = [];
+        // Iterate over each row in the worksheet
+        while (parser.next()) {
+            row = parser.getRow();
+            gs.info("Shamma row" +headers.length);
```
Comment (ashu759) [inline Flow Actions/Excel Parser/Excel Parser.js:24]: Clean up the code please. Like remove comments and logs file. Clean code is required.

**Review by ashu759 — COMMENTED** [review]
Also, i read what you have mentioned in the readme file but that all does not reflect in your code. I think there a better ways to update and create records in ServiceNow like Transform maps.

**Comment by spnegi** [comment]
Sure let me remove the comments and clean the code and let you know.

We can also set the parent child relationship via excel parser.

**Comment by spnegi** [comment]
Hi @ashu759 , 

I have removed the comments and info log lines from the code. Also updated the readme.md.
Please verify now

Thanks,
Shamma

**Comment by ashu759** [comment]
Hello @spnegi , When we run duplicate check, we already see that this functionality was already committed https://github.com/ServiceNowDevProgram/code-snippets/pull/102. So for now we are closing this pull request as we dont see much value addition in this.

**Pull request closed without merging**